timApp/static/stylesheets/velpSelection.scss

```diff
@@ -68,8 +88,8 @@ annotation .fulldiv {
 /** Velp area styles **/
 .velpMenu {
     /* position: fixed; */
-    min-width: fit-content;
-    max-width: 23em;
+    //min-width: fit-content;
+    //max-width: 23em;
```

Comment:
Turhat kommentit pois. Alempana näitä on lisää, en kommentoi niitä.

Reply:
Poistettu turhat